### PR TITLE
docs: add three Chinese version about Authentication doc

### PR DESCRIPTION
**Pull request by SylviaBABY**
### Description

In the current Chinese document, these three articles have no relevant content, now will add the content.

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/ldap-auth.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+## 描述
+
+`ldap-auth`是用来给路由或服务添加 LDAP 认证的插件，使用 [lualdap](https://lualdap.github.io/lualdap/)连接 LDAP 服务器。
+
+该插件需要与 `consumer` 一起配合使用，API Consumer 可以使用 [basic authentication](https://en.wikipedia.org/wiki/Basic_access_authentication)与 LDAP 服务器进行认证。
+
+## 属性
+
+Consumer 端配置：
```
Comment (hf400159) [inline docs/zh/latest/plugins/ldap-auth.md:38]:
```suggestion
Consumer 端：
```

File: docs/zh/latest/plugins/ldap-auth.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+| 名称    | 类型   | 必选项 | 描述                                                                      |
+| ------- | ------ | -------- | -------------------------------------------------------------------------------- |
+| user_dn | string | 是     | LDAP 客户端的用户可分辨名称，例如：`cn=user01,ou=users,dc=example,dc=org`。 |
+
+Route 端配置：
```
Comment (hf400159) [inline docs/zh/latest/plugins/ldap-auth.md:44]:
```suggestion
Route 端：
```

File: docs/zh/latest/plugins/ldap-auth.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+| 名称     | 类型    | 必选项 | 默认值 | 描述                                                            |
+|----------|---------|----------|---------|------------------------------------------------------------------------|
+| base_dn  | string  | 是     |         | LDAP 服务器的基础可分辨名称，例如：`ou=users,dc=example,dc=org`。|
+| ldap_uri | string  | 是     |         | LDAP 服务器的 URI。                                                |
+| use_tls  | boolean | 否    | `true`  | 如果设置为 `true` 则表示启用 TLS。                                             |
+| uid      | string  | 否    | `cn`    | UID 属性。                                                         |
```
Comment (hf400159) [inline docs/zh/latest/plugins/ldap-auth.md:51]:
```suggestion
| uid      | string  | 否    | cn    | UID 属性。                                                         |
```

File: docs/zh/latest/plugins/ldap-auth.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+        "nodes": {
+            "127.0.0.1:1980": 1
+        }
+    }
+}'
+```
+
+## 测试插件
+
+按照上文所述进行插件配置后，用户就可以通过授权提出请求并访问 API：
```
Comment (hf400159) [inline docs/zh/latest/plugins/ldap-auth.md:94]:
```suggestion
通过上述方法配置插件后，可以通过以下命令测试插件：
```

File: docs/zh/latest/plugins/ldap-auth.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+    "username": "foo",
+    "plugins": {
+        "ldap-auth": {
+            "user_dn": "cn=user01,ou=users,dc=example,dc=org"
+        }
+    }
+}'
+```
+
+然后你就可以在指定 Route 或 Service 中启用该插件，具体代码如下：
```
Comment (hf400159) [inline docs/zh/latest/plugins/ldap-auth.md:69]:
```suggestion
然后你就可以在指定 Route 或 Service 中启用该插件，具体代码如下：
```

File: docs/zh/latest/plugins/opa.md [hunk 6]
```diff
@@ -0,0 +1,315 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 描述
+
+`opa` 插件可用于与 [Open Policy Agent](https://www.openpolicyagent.org) 进行集成，实现后端服务的认证授权和访问服务等功能解耦，减少系统复杂性。
```
Comment (hf400159) [inline docs/zh/latest/plugins/opa.md:32]:
```suggestion
`opa` 插件可用于与 [Open Policy Agent](https://www.openpolicyagent.org) 进行集成，实现后端服务的认证授权与访问服务等功能解耦，减少系统复杂性。
```

File: docs/zh/latest/plugins/opa.md [hunk 7]
```diff
@@ -0,0 +1,315 @@
...
+        "remote_port": "port",
+        "remote_addr": "ip address"
+    },
+    "route": {},
+    "service": {},
+    "consumer": {}
+}
+```
+
+上述代码中释义如下：
```
Comment (hf400159) [inline docs/zh/latest/plugins/opa.md:84]:
```suggestion
上述代码具体释义如下：
```

File: docs/zh/latest/plugins/opa.md [hunk 8]
```diff
@@ -0,0 +1,315 @@
...
+    "upstream": {
+        "nodes": {
+            "httpbin.org:80": 1
+        },
+        "type": "roundrobin"
+    }
+}'
+```
+
+之后可对其进行如下测试：
```
Comment (hf400159) [inline docs/zh/latest/plugins/opa.md:163]:
```suggestion
使用如下命令进行测试：
```

File: docs/zh/latest/plugins/opa.md [hunk 9]
```diff
@@ -0,0 +1,315 @@
...
+
+## 属性
+
+| 名称              | 类型    | 必选项 | 默认值 | 有效值  | 描述                                                                                                                                                                                |
+|-------------------|---------|----------|---------|---------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| host              | string  | 是     |         |               | OPA 服务的主机地址，例如 `https://localhost:8181`。                                                                                                                   |
+| ssl_verify        | boolean | 否    | true    |               | 当设置为 `true` 时，将验证 SSL 证书。                                                                                                                                          |
+| policy            | string  | 是     |         |               | OPA 政策路径，是 `package` 和 `decision` 配置的组合。当使用高级功能如自定义响应时，你可以省略`decision` 配置。                                                  |
```
Comment (hf400159) [inline docs/zh/latest/plugins/opa.md:40]:
```suggestion
| policy            | string  | 是     |         |               | OPA 策略路径，是 `package` 和 `decision` 配置的组合。当使用高级功能（例如自定义响应）时，你可以省略 `decision` 配置。                                                  |
```

File: docs/zh/latest/plugins/opa.md [hunk 10]
```diff
@@ -0,0 +1,315 @@
...
+| timeout           | integer | 否    | 3000ms  | [1, 60000]ms  | HTTP 调用延迟设置。                                                                                                                                                                |
```
Comment (hf400159) [inline docs/zh/latest/plugins/opa.md:41]:
```suggestion
| timeout           | integer | 否    | 3000ms  | [1, 60000]ms  | 设置 HTTP 调用超时时间。                                                                                                                                                                |
```

File: docs/zh/latest/plugins/opa.md [hunk 11]
```diff
@@ -0,0 +1,315 @@
...
+| keepalive         | boolean | 否    | true    |               | 当设置为 `true` 时，将为多个请求保持连接状态。                                                                                                                               |
```
Comment (hf400159) [inline docs/zh/latest/plugins/opa.md:42]:
```suggestion
| keepalive         | boolean | 否    | true    |               | 当设置为 `true` 时，将为多个请求保持连接处于活动状态。                                                                                                                               |
```

File: docs/zh/latest/plugins/opa.md [hunk 12]
```diff
@@ -0,0 +1,315 @@
...
+| keepalive_timeout | integer | 否    | 60000ms | [1000, ...]ms | 连接断开后的闲置时间。                                                                                                                                                                        |
+| keepalive_pool    | integer | 否    | 5       | [1, ...]ms    | 连接池限制。                                                                                                                                                                    |
+| with_route        | boolean | 否    | false   |               | 当设置为 `true` 时，发送关于当前 Route 的信息。                                                                                                                              |
+| with_service      | boolean | 否    | false   |               | 当设置为 `true` 时，发送关于当前 Service 的信息。                                                                                                                            |
+| with_consumer     | boolean | 否    | false   |               | 当设置为 `true` 时，发送关于当前 Consumer 的信息。注意，这可能会发送敏感信息，如 API 密钥。确保只有在你确定安全的情况下才打开它。 |
```
Comment (hf400159) [inline docs/zh/latest/plugins/opa.md:47]:
```suggestion
| with_consumer     | boolean | 否    | false   |               | 当设置为 `true` 时，发送关于当前 Consumer 的信息。注意，这可能会发送敏感信息，如 API key。请确保在安全的情况下打开它。 |
```

File: docs/zh/latest/plugins/opa.md [hunk 13]
```diff
@@ -0,0 +1,315 @@
...
+    "consumer": {}
+}
+```
+
+上述代码具体释义如下：
+
+- `type` 代表请求类型（如 `http` 或 `stream`）；
+- `request` 则需要在 `type` 为 `http` 时使用，包含基本的请求信息（如 URL、头信息等）；
+- `var` 包含关于请求连接的基本信息（如 IP、端口、请求时间戳等）；
+- `route`、`service` 和 `consumer` 会包含与存储在 APISIX 中相同的数据，只有当 `opa` 插件配置在这些对象上时才会发送。
```
Comment (hf400159) [inline docs/zh/latest/plugins/opa.md:89]:
```suggestion
- `route`、`service` 和 `consumer` 包含的数据，与 APISIX 中存储的数据相同，只有当这些对象上配置了 `opa` 插件时才会发送。
```

File: docs/zh/latest/plugins/opa.md [hunk 14]
```diff
@@ -0,0 +1,315 @@
...
+
+### 场景二：OPA 向 APISIX 发送数据
+
+下述示例代码展示了如何通过 OPA 服务向 APISIX 发送数据：
```
Comment (hf400159) [inline docs/zh/latest/plugins/opa.md:93]:
```suggestion
下述示例代码展示了 OPA 服务对 APISIX 发起请求后的响应数据：
```

File: docs/zh/latest/plugins/opa.md [hunk 15]
```diff
@@ -0,0 +1,315 @@
...
+        "reason": "test",
+        "headers": {
+            "an": "header"
+        },
+        "status_code": 401
+    }
+}
+```
+
+上述代码响应中释义如下：
```
Comment (hf400159) [inline docs/zh/latest/plugins/opa.md:108]:
```suggestion
上述响应中的代码释义如下：
```

File: docs/zh/latest/plugins/opa.md [hunk 16]
```diff
@@ -0,0 +1,315 @@
...
+
+default allow = false
+
+allow {
+    # HTTP method must GET
+    request.method == "GET"
+}'
+```
+
+然后在一个特定的 Route 上配置 `opa` 插件：
```
Comment (hf400159) [inline docs/zh/latest/plugins/opa.md:140]:
```suggestion
然后在指定路由上配置 `opa` 插件：
```

File: docs/zh/latest/plugins/opa.md [hunk 17]
```diff
@@ -0,0 +1,315 @@
...
+
+```shell
+curl -i -X GET 127.0.0.1:9080/get
+```
+
+```shell
+HTTP/1.1 200 OK
+```
+
+这时如果你发出一个失败请求，将会收到来自 OPA 服务的自定义响应反馈，如下所示：
```
Comment (hf400159) [inline docs/zh/latest/plugins/opa.md:228]:
```suggestion
此时如果你发出一个失败请求，将会收到来自 OPA 服务的自定义响应反馈，如下所示：
```

File: docs/zh/latest/plugins/opa.md [hunk 18]
```diff
@@ -0,0 +1,315 @@
...
+```shell
+curl -X PUT '127.0.0.1:8181/v1/policies/echo' \
+    -H 'Content-Type: text/plain' \
+    -d 'package echo
+
+allow = false
+reason = input'
+```
+
+现在就可以在 Route 上配置插件来发送 APISIX 数据：
```
Comment (hf400159) [inline docs/zh/latest/plugins/opa.md:256]:
```suggestion
现在就可以在路由上配置插件来发送 APISIX 数据：
```

File: docs/zh/latest/plugins/opa.md [hunk 19]
```diff
@@ -0,0 +1,315 @@
...
+    "upstream": {
+        "nodes": {
+            "httpbin.org:80": 1
+        },
+        "type": "roundrobin"
+    }
+}'
+```
+
+此时如果你提出一个请求，则可以通过自定义响应看到来自 Route 的数据：
```
Comment (hf400159) [inline docs/zh/latest/plugins/opa.md:280]:
```suggestion
此时如果你发起一个请求，则可以通过自定义响应看到来自路由的数据：
```

File: docs/zh/latest/plugins/opa.md [hunk 20]
```diff
@@ -0,0 +1,315 @@
...
+
+```shell
+curl -X GET 127.0.0.1:9080/get
```
Comment (hf400159) [inline docs/zh/latest/plugins/opa.md:283]: Please split into two code blocks

File: docs/en/latest/plugins/authz-casdoor.md [hunk 21]
```diff
@@ -85,3 +85,22 @@ Once you have enabled the Plugin, a new user visiting this Route would first be
...
+
+## Disable Plugin
+
+To disable the `authz-casdoor` Plugin, you can delete the corresponding JSON configuration from the Plugin configuration. APISIX will automatically reload and you do not have to restart for this to take effect.
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "methods": ["GET"],
+    "uri": "/anything/*",
```
Comment (juzhiyuan) [inline docs/en/latest/plugins/authz-casdoor.md:97]: One suggestion: we'd better use a more straightforward URI instead of `/anything`, for people would get confused if they don't know this is from `httpbin.org/anything`.

(No need to do this in this PR)

File: docs/zh/latest/plugins/authz-casdoor.md [hunk 22]
```diff
@@ -0,0 +1,102 @@
...
+
+使用 `authz-casdoor` 插件可添加 [Casdoor](https://casdoor.org/) 集中认证方式。
+
+## 属性
+
+| 名称          | 类型   | 必选项 | 描述                                  |
+|---------------|--------|----------|----------------------------------------------|
+| endpoint_addr | string | 是     | Casdoor 的 URL。                           |
+| client_id     | string | 是     | Casdoor 的客户端 id。                      |
+| client_secret | string | 是     | Casdoor 的客户端密码。                 |
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/authz-casdoor.md:40]:
```suggestion
| client_secret | string | 是     | Casdoor 的客户端密钥。                 |
```

File: docs/zh/latest/plugins/authz-casdoor.md [hunk 23]
```diff
@@ -0,0 +1,102 @@
...
+| callback_url  | string | 是     | 用于接收状态和代码的回调 URL。 |
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/authz-casdoor.md:41]:
```suggestion
| callback_url  | string | 是     | 用于接收 code 与 state 的回调地址。 |
```

File: docs/zh/latest/plugins/authz-casdoor.md [hunk 24]
```diff
@@ -0,0 +1,102 @@
...
+    }
+  }
+}'
+```
+
+## 测试插件
+
+一旦启用了该插件，访问该路由的新用户首先会经过 `authz-casdoor` 插件的处理，然后被重定向到 Casdoor 登录页面。
+
+成功登录后，Casdoor 会将该用户重定向到 `callback_url`，并指定 GET 参数的 `code` 和 `state`。该插件还会向 Casdoor 请求一个访问 Token，并确认用户是否真正登录了。该过程只运行一次，后续请求将不会被此打断。
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/authz-casdoor.md:81]:
```suggestion
成功登录后，Casdoor 会将该用户重定向到 `callback_url`，并指定 GET 参数的 `code` 和 `state`。该插件还会向 Casdoor 请求一个访问 Token，并确认用户是否已登录。在成功认证后该流程只出现一次，后续请求不会被打断。
```

File: docs/zh/latest/plugins/authz-casdoor.md [hunk 25]
```diff
@@ -0,0 +1,102 @@
...
+
+上述操作完成后，用户就会被重定向到他们本想访问的原始 URL 页面。
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/authz-casdoor.md:83]:
```suggestion
上述操作完成后，用户会被重定向到目标 URL。
```

File: docs/zh/latest/plugins/ldap-auth.md [hunk 26]
```diff
@@ -0,0 +1,155 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 描述
+
+`ldap-auth` 插件可用于给路由或服务添加 LDAP 身份认证，该插件使用 [lualdap](https://lualdap.github.io/lualdap/) 连接 LDAP 服务器。
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/ldap-auth.md:32]:
```suggestion
`ldap-auth` 插件可用于给路由（Route）或服务（Service）添加 LDAP 身份认证，该插件使用 [lualdap](https://lualdap.github.io/lualdap/) 连接 LDAP 服务器。
```

File: docs/zh/latest/plugins/ldap-auth.md [hunk 27]
```diff
@@ -0,0 +1,155 @@
...
+
+该插件需要与 Consumer 一起配合使用，API 的调用方可以使用 [basic authentication](https://en.wikipedia.org/wiki/Basic_access_authentication) 与 LDAP 服务器进行认证。
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/ldap-auth.md:34]: Could we use Docusaurus's mark like Info or Warn?

File: docs/zh/latest/plugins/ldap-auth.md [hunk 28]
```diff
@@ -0,0 +1,155 @@
...
+    "username": "foo",
+    "plugins": {
+        "ldap-auth": {
+            "user_dn": "cn=user01,ou=users,dc=example,dc=org"
+        }
+    }
+}'
+```
+
+然后就可以在指定 Route 或 Service 中启用该插件，具体代码如下：
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/ldap-auth.md:69]: We should have a unified style: use Route/Service or `路由/服务` or `Route（路由）`

File: docs/zh/latest/plugins/opa.md [hunk 29]
```diff
@@ -0,0 +1,318 @@
+---
+title: opa
+keywords:
+  - APISIX
+  - Plugin
+  - Open Policy Agent
+  - opa
+description: 本篇文档介绍了 Apache APISIX opa 插件的相关信息。
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/opa.md:8]:
```suggestion
description: 本篇文档介绍了 Apache APISIX 通过 OPA 插件与 Open Policy Agent 对接的相关信息。
```

File: docs/zh/latest/plugins/opa.md [hunk 30]
```diff
@@ -0,0 +1,318 @@
+---
+title: opa
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/opa.md:2]:
```suggestion
title: Open Policy Agent (OPA)
```

File: docs/zh/latest/plugins/opa.md [hunk 31]
```diff
@@ -0,0 +1,318 @@
...
+- `var` 包含关于请求连接的基本信息（如 IP、端口、请求时间戳等）；
+- `route`、`service` 和 `consumer` 包含的数据与 APISIX 中存储的数据相同，只有当这些对象上配置了 `opa` 插件时才会发送。
+
+### OPA 向 APISIX 返回数据
+
+下述示例代码展示了 OPA 服务对 APISIX 发送请求后的响应数据：
+
+```json
+{
+    "result": {
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/opa.md:97]: 2 spaces

File: docs/zh/latest/plugins/opa.md [hunk 32]
```diff
@@ -0,0 +1,318 @@
...
+
+## 数据定义
+
+### APISIX 向 OPA 发送信息
+
+下述示例代码展示了如何通过 APISIX 向 OPA 服务发送数据：
+
+```json
+{
+    "type": "http",
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/opa.md:57]: We'd better keep 2 spaces for JSON (Could do this in a separate PR)

File: docs/zh/latest/plugins/ldap-auth.md (hunk 26 above)
Comment (SylviaBABY) [reply docs/zh/latest/plugins/ldap-auth.md:32]: I think we could present the Chinese definitions of these terms in the Chinese documents, and I will check them in other documents

File: docs/zh/latest/plugins/ldap-auth.md (hunk 27 above)
Comment (SylviaBABY) [reply docs/zh/latest/plugins/ldap-auth.md:34]: We could, but such `jwt-auth` and `key-auth` docs have the same description without INFO MARK, so maybe we can let these contents present the same temporarily, and then adjust the whole thing together if we want to change it.

File: docs/en/latest/plugins/authz-casdoor.md (hunk 21 above)
Comment (SylviaBABY) [reply docs/en/latest/plugins/authz-casdoor.md:97]: Got it, will attention to this point later

File: docs/zh/latest/plugins/ldap-auth.md (hunk 27 above)
Comment (juzhiyuan) [reply docs/zh/latest/plugins/ldap-auth.md:34]: ok

File: docs/zh/latest/plugins/ldap-auth.md (hunk 26 above)
Comment (juzhiyuan) [reply docs/zh/latest/plugins/ldap-auth.md:32]: ok

**Review by juzhiyuan — DISMISSED** [review]

File: docs/zh/latest/plugins/opa.md (hunk 30 above)
Comment (SylviaBABY) [reply docs/zh/latest/plugins/opa.md]: I found that the name of the title is displayed in the sidebar, because the current sidebar display is the name of the plugin, so this should be displayed as opa more appropriate.

Comment (juzhiyuan) [reply docs/zh/latest/plugins/opa.md:2]: Just have a look at OPA's website https://www.openpolicyagent.org/

I noticed they are using OPA as a short name, too. LGTM

**Review by juzhiyuan — APPROVED** [review]

**Review by Baoyuantop — APPROVED** [review]
